packages/assets-controllers/src/TokenRatesController.test.ts

```diff
@@ -127,7 +128,12 @@ describe('TokenRatesController', () => {
             allTokens: {
               '0x1': {
                 [defaultSelectedAddress]: [
-                  { address: 'bar', decimals: 0, symbol: '', aggregators: [] },
+                  {
+                    address: mockTokenAddress,
```

Comment:
I had to update a bunch of test fixtures to use hex addresses rather than non-hex strings, as it threw an error when they were passed to `toHex`